src/ShearStressTransportEquationSystem.C

```diff
@@ -554,5 +578,64 @@ ShearStressTransportEquationSystem::compute_f_one_blending()
   }
 }
 
+//------------------------------------------------------------------------------
```

Comment:
In general, I think we are going to want to throw all the code into a whitespace cleanup (if you use emacs/vi?). Or can you try to get `clang-format` to work? That might be best.

Reply:
clang-format? please, can you expand? thanks

Reply:
It is a tool is affiliated with Clang (https://clang.llvm.org/docs/ClangFormat.html). You send your code through it and it will automatically reformat it to follow certain rules.  It can be integrated with various editors (emacs/vim/visual studio) and do it automatically for you. Don't waste too much time on it (one of us can do it if need be, but it does make it easier for other people reading your code.